main/src/main/java/org/mobilitydata/gtfsvalidator/notice/AttributionWithoutRoleNotice.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A row from GTFS file `attributions.txt` has fields `attributions.is_producer`,
+ * `attributions.is_operator`, and `attributions.is_authority` not defined or set at 0.
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
+ */
+public class AttributionWithoutRoleNotice extends ValidationNotice {
+  public AttributionWithoutRoleNotice(long csvRowNumber) {
+    super(
+        new ImmutableMap.Builder<String, Object>().put("csvRowNumber", csvRowNumber).build(),
```

Comment:
This may be written in a shorter and more idiomatic way: ImmutableMap.of(csvRowNumber", csvRowNumber).

Please may I ask the reviewers to pay attention to idiomaticity of the code?

Please may I ask the code author to take a look at similar Notice classes and follow the existing practices?